htdocs/compta/paiement.php

```diff
@@ -456,12 +457,15 @@ function callForResult(imgId)
 		print '<input type="hidden" name="type" id="invoice_type" value="'.$facture->type.'">';
 		print '<input type="hidden" name="thirdpartylabel" id="thirdpartylabel" value="'.dol_escape_htmltag($facture->thirdparty->name).'">';
 
-		print dol_get_fiche_head();
 
-		print '<table class="border centpercent">';
+		print '<div class="fichecenter">';
+		print '<div class="fichehalfleft">';
+		print '<div class="underbanner clearboth"></div>';
+
+		print '<table class="noborder tableforfield" centpercent>';
 
 		// Third party
-		print '<tr><td class="titlefieldcreate"><span class="fieldrequired">'.$langs->trans('Company').'</span></td><td>'.$facture->thirdparty->getNomUrl(4)."</td></tr>\n";
+		print '<tr><td class="width25p fieldrequired">'.$langs->trans('Company').'</td><td>'.$facture->thirdparty->getNomUrl(4)."</td></tr>\n";
```

Comment:
Why replacing the css titlefieldcreate used by all forms to defined the width of first column ?

Reply:
because the column is too small (we have carriage return on other fields line)